autotst/job/job.py

```diff
@@ -540,87 +705,257 @@ def calculate_transitionstate(self, transitionstate, vibrational_analysis=True):
             global_results[ts_identifier] = False
             return False
 
-    def calculate_reaction(self, vibrational_analysis=True):
+    def check_irc_folder(self, reaction):
+        """
+        A method to check irc jobs to try to find an irc validated transition state
+        Returns true if a validated ts is found, false otherwise
+        """
+        got_one = False
+        irc_folder = os.path.join(
+            self.directory, "ts", reaction.label, 'irc')
+        if not os.path.exists(irc_folder):
+            logging.info("It appears no irc jobs were run")
+            return False
+        else:
+            irc_logs = [l for l in os.listdir(irc_folder) if l.endswith('.log')]
+            for log in irc_logs:
+                if 'forward' in log:
+                    direction = 'forward'
+                else:
+                    direction = 'reverse'
+                ts = reaction.ts[direction][0]
+                ts.index = int(log.split('.log')[0].split('_')[-1])
+                label = ts.reaction_label + '_' + direction + '_' + str(ts.index)
+                self.calculator.conformer = ts
+                result = self.calculator.validate_irc()
+                if result:
+                    logging.info("{} successfully validated via irc!".format(label))
+                    if os.path.exists(self.directory, "ts", reaction.label, "conformers",label+'.log'):
+                        got_one=True
+                        copyfile(os.path.join(self.directory, "ts", self.reaction.label,"conformers", label +'.log'),
+                                os.path.join(self.directory, "ts", self.reaction.label, self.reaction.label + ".log")
+                                )
+                        logging.info("TS log file validated from irc is {} :)".format(
+                        os.path.join(self.directory, "ts",self.reaction.label, self.reaction.label + ".log")))
```

Comment:
Possibly store a list of successful and failed IRC files. Also, capitalize IRC in logging plz